operator/src/lib.rs

```diff
@@ -331,22 +352,20 @@ impl ControllerStrategy for KafkaStrategy {
         let cluster_spec: KafkaClusterSpec = context.resource.spec.clone();
 
         let mut eligible_nodes = HashMap::new();
+        let test = cluster_spec
```

Comment:
Maybe replace the variable "test" with "role_groups" or similiar?

Reply:
done, that was leftover from development